packages/zowe-explorer/src/utils/LocalFileManagement.ts

```diff
@@ -10,119 +10,50 @@
-import { ZoweDatasetNode } from "../dataset/ZoweDatasetNode";
-import { ZoweUSSNode } from "../uss/ZoweUSSNode";
 
 export class LocalFileManagement {
+    public static filesToCompare: IZoweTreeNode[] = [];
+    public static fileSelectedToCompare: boolean = false;
+
     public static getDefaultUri(): vscode.Uri {
         return vscode.workspace.workspaceFolders?.[0]?.uri ?? vscode.Uri.file(os.homedir());
     }
-    /**
-     * Function that triggers compare of the old and new document in the active editor
-     * @param {vscode.TextDocument} doc - document to update and compare with previous content
-     * @param {IZoweDatasetTreeNode | IZoweUSSTreeNode} node - IZoweTreeNode
-     * @param {string} label - {optional} used by IZoweDatasetTreeNode to getContents of file
-     * @param {boolean} binary - {optional} used by IZoweUSSTreeNode to getContents of file
-     * @param {imperative.IProfileLoaded} profile - {optional}
-     * @returns {Promise<void>}
-     */
-    public static async compareSavedFileContent(
-        doc: vscode.TextDocument,
-        node: IZoweDatasetTreeNode | IZoweUSSTreeNode,
-        label?: string,
-        profile?: imperative.IProfileLoaded
-    ): Promise<void> {
-        await markDocumentUnsaved(doc);
-        const prof = node ? node.getProfile() : profile;
-        let downloadResponse;
 
-        if (isTypeUssTreeNode(node)) {
-            downloadResponse = await ZoweExplorerApiRegister.getUssApi(prof).getContents(node.fullPath, {
-                file: node.getUSSDocumentFilePath(),
-                binary: node.binary,
-                returnEtag: true,
-                encoding: node.encoding !== undefined ? node.encoding : prof.profile?.encoding,
-                responseTimeout: prof.profile?.responseTimeout,
-            });
-        } else {
-            downloadResponse = await ZoweExplorerApiRegister.getMvsApi(prof).getContents(label, {
-                file: doc.fileName,
-                binary: node.binary,
-                returnEtag: true,
-                encoding: node.encoding !== undefined ? node.encoding : prof.profile?.encoding,
-                responseTimeout: prof.profile?.responseTimeout,
-            });
-        }
-        ZoweLogger.warn(vscode.l10n.t("Remote file has changed. Presenting with way to resolve file."));
-        vscode.commands.executeCommand("workbench.files.action.compareWithSaved");
-        // re-assign etag, so that it can be used with subsequent requests
-        const downloadEtag = downloadResponse?.apiResponse?.etag;
-        if (node && downloadEtag !== node.getEtag()) {
-            node.setEtag(downloadEtag);
-        }
+    public static setCompareSelection(val: boolean): void {
+        LocalFileManagement.fileSelectedToCompare = val;
+        vscode.commands.executeCommand("setContext", "zowe.compareFileStarted", val);
+    }
+
+    public static reset(): void {
```

Comment:
Should we rename this to be more specific as to what we are resetting? (maybe `resetCompareSelection`) 😋